### PR TITLE
Possibility to pass the command that will be executed on the container

### PR DESCRIPTION
**Pull request by robertogyn19**
This scenario is useful in cases where you need to pass some parameter that is not via environment variable.

### CODE REVIEW

**Comment by aeneasr** [comment]
Hey, thanks, this looks solid!

File: dockertest.go [hunk 1]
```diff
@@ -133,6 +146,19 @@ func (d *Pool) Run(repository, tag string, env []string) (*Resource, error) {
...
+//
+//  pool.Run("mysql", "5.3", []string{"FOO=BAR", "BAR=BAZ"})
+func (d *Pool) Run(repository, tag string, env []string) (*Resource, error) {
+	opts := RunOptions{
+		Repository: repository,
+		Tag:        tag,
+		Env:        env,
+	}
+
+	return d.RunWithOptions(opts)
```
Comment (aeneasr) [inline dockertest.go]: This could probably be a one liner:

```
return d.RunWithOptions(RunOptions{})
```

File: dockertest_test.go [hunk 2]
```diff
@@ -55,3 +56,22 @@ func TestPostgres(t *testing.T) {
...
+		Tag:        "3.3.12",
+		Cmd:        []string{"mongod", "--smallfiles"},
+	}
+	resource, err := pool.RunWithOptions(options)
+	require.Nil(t, err)
+	port := resource.GetPort("27017/tcp")
+	assert.NotEmpty(t, port)
+
+	err = pool.Retry(func() error {
+		_, err := http.Get(fmt.Sprintf("http://127.0.0.1:%s", port))
```
Comment (aeneasr) [inline dockertest_test.go]: Also check for status code here, just to be sure?

**Comment by robertogyn19** [comment]
Thanks for the comments, I've already made the suggested changes.

**Comment by aeneasr** [comment]
Thanks!

**Pull request merged**